docs/rfcs/015-iiif-av-delivery-channel-settings.md

```diff
@@ -0,0 +1,132 @@
+# How are iiif-av settings going to be maintained in protagonist
+
+
+## Problem
+
+https://deploy-preview-2--dlcs-docs.netlify.app/api-doc/delivery-channels#iiif-av
+
+We decided to use named _DLCS_ presets as the policyData.
+
+```
+   "policyData": "[ \"video-mp4-720p\" ]",
+```
+
+When Engine is given an asset with:
+
+```json
+  "origin":  "s3://my-bucket/video-masters/huge-video.mpeg",
+  "deliveryChannels": [
+    {
+      "@type": "vocab:DeliveryChannel",
+      "channel": "iiif-av",
+      "policy": "https://api.dlcs.io/customers/2/deliveryChannelPolicies/iiif-av/standard"
+    }
+  ]
+```
+
+... and `https://api.dlcs.io/customers/2/deliveryChannelPolicies/iiif-av/standard` looks like this:
+
+```json
+{
+     "@id": "https://api.dlcs.io/customers/2/deliveryChannelPolicies/iiif-av/standard",
+     "@type": "vocab:DeliveryChannelPolicy",
+     "displayName": "Default video transcode",
+     "channel": "iiif-av",
+     "policyData":  "[ \"video-mp4-720p\" ]",
+     "policyModified": "2023-09-19T15:36:58.6023600Z"
+ }
+```
+
+...then Engine _looks up_ "video-mp4-720p" and sees, in some form, "Transcode with "System preset: 'Mp4 HLS 720p'; Extension: 'mp4'" so it calls Elastic Transcoder with that information.

```

Comment:
Use backticks for readability? `"Transcode with "System preset: 'Mp4 HLS 720p'; Extension: 'mp4'"`